pedpy/methods/method_utils.py

```diff
@@ -44,6 +48,14 @@ class SpeedCalculation(Enum):  # pylint: disable=too-few-public-methods
     BORDER_SINGLE_SIDED = auto()
 
 
+class ReturnCode(Enum):  # pylint: disable=too-few-public-methods
```

Comment:
Please use a more specific name for this. A return code, could be used everywhere

Reply:
renamed to `DataValidationStatus`